stdlib/Makefile

```diff
@@ -237,13 +236,16 @@ include .depend
 EMPTY :=
 SPACE := $(EMPTY) $(EMPTY)
 
+STDLIB_NAMESPACE_MODULES = $(subst $(SPACE),|,$(STDLIB_PREFIXED_MODULES))
+
 .PHONY: depend
 depend:
 	$(CAMLDEP) $(DEPFLAGS) $(filter-out stdlib.%,$(wildcard *.mli *.ml)) \
 	  > .depend.tmp
 	$(CAMLDEP) $(DEPFLAGS) -pp "$(AWK) -f ./remove_module_aliases.awk" \
 	  stdlib.ml stdlib.mli >> .depend.tmp
-	sed -Ee \
-	  's#(^| )(${subst ${SPACE},|,${UNPREFIXED_OBJS}})[.]#\1stdlib__\2.#g' \
+	sed -E \
+	-e 's#(^| )(${STDLIB_NAMESPACE_MODULES})[.]#\1stdlib__\u\2.#g' \
+	-e 's/^stdlib__(.)(.*)(\.[^i]*)(i?) :/stdlib__\1\2\3\4 : \l\1\2.ml\4/' \
```

Comment:
Note: here is a rough idea for an alternative:
- first add the `.mli` files as first dependencies (this requires no case-change magic)
- then *generate* the rest of the sed script by iterating on STDLIB_NAMESPACE_MODULES and producing a textual replacement instead (or we could, if we fill less meta-programming-y, just use a shell loop with `sed -i`).

I will let you judge whether this can work and is better than the current approach.

Reply:
Not sure (yet) about the loop, but changing the order of those two expressions is definitely a good idea, if only to remove the evil of `\l\1` which certainly in the fixed fonts in my editor/browser is not immediately clear to be a lowercase "l" and a "1"!

Reply:
The rebased version swaps the two statements, which simplifies the second one quite considerably